src/components/editable/Editable.js

```diff
@@ -0,0 +1,41 @@
+import './editable.scss';
+
+class Editable extends Component {
+  state = {
+    editable: false,
+    val: 'Something',
+
+  }
+
+  handleClick = (event) => {
+    const { callback } = this.props;
+    this.setState(prev => ({ editable: !prev.editable }));
+    callback(event.target.value);
```

Comment:
1. you should check whether `callback` was passed from props
2. no need to call `callback` if user clicks on span

Reply:
fixed issues in next commit